tests/Feature/CommentApiControllerTest.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Tests\Unit;
+
+use App\Http\Controllers\Api\CommentController;
+use App\Models\Comment;
+use App\Models\HrAdvisor;
+use App\Models\JobPoster;
+use App\Models\Manager;
+use Tests\TestCase;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Http\Resources\Json\JsonResource;
+use Illuminate\Http\Resources\Json\ResourceCollection;
+use Illuminate\Support\Facades\Log;
+
+class CommentApiControllerTest extends TestCase
```

Comment:
I just realized this class is called CommentApiControllerTest, but the class its testing is actually just CommentController. 😛